app/forms/candidate_interface/degree_type_form.rb

```diff
@@ -89,5 +91,13 @@ def international?
     def uk?
       uk_degree == 'yes'
     end
+
+    def degree_type_uuid
+      new_data_degree_type = DfE::ReferenceData::Degrees::TYPES_INCLUDING_GENERICS.all.find do |degree_type|
+        degree_type.name == type_description
```

Comment:
also synonyms here?

Reply:
The types almost don't have synonyms in the new data. But in the next PR, I am searching for synonyms as well for types.